### ISSUE TITLE
Research: Establish contact info availability + best channel(s) for outreach

### ISSUE THREAD

**Issue by aplatt-coforma** [department-of-veterans-affairs/va.gov-team#66430]
## Goals
Determine best ways to contact target cohort

Unclear how many Veterans with active diagnoses can be reliably contacted - (VA Notify) email is 'easiest'. Other channels like paper, text etc exist.

## To Do
- [ ] Understand overall veteran experience on receiving end with the different communications they get (related to Emily, Danielle conversations)
- [ ] Establish channel for POC

**Comment by aplatt-coforma** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/benefits-proactive-64e7650c29bb1113de04a02a/issues/department-of-veterans-affairs/va.gov-team/66430?planning-poker) with Zenhub @gmaciolek @djones2113

**Comment by jenniferbertsch** [comment]
Hi - this one is also in the Mural list for sprint 6 but maybe blocked by user research?

**Comment by djones2113** [comment]
Yes! We should have more information to move forward after user research. To provide some specifics, we're aiming to run research during Sprint 7 (between 11/14-11/21), with synthesis taking place the follow week or so in Sprint 8.

**Comment by djones2113** [comment]
@aplatt-coforma is it possible to make this ticket, as well as #67981, sub-tickets of #70125? Or indicate that both of these tickets are blocked by #70125? These will be tickets that will be addressed (at least initially) via the research outcomes and recommendations.

@gmaciolek let me know if you have any objections to this or feel differently!

**Comment by djones2113** [comment]
What we found in our research was that a strong case wasn’t made to either avoid or pursue a specific channel for outreach.

A letter was in line with most participants' expectations for this type of outreach, but no one indicated mail as their preferred form of communication. There was recurring sentiment that mail is slow and unreliable, not only when it comes to receiving mail but also sending it. Some participants expressed concerns with filing via mail when reviewing filing options in our outreach.

Our interviews ultimately focused more on content than channel, and we hypothesize that the content of the outreach will be more impactful in driving action than the channel of the outreach itself.

For detailed information on this finding, refer to the first section (Goal 1) of the [research readout](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1699309433965/72916a05e4498e90ada15f87aa4e66ddf8e7d959?wid=0-1703010739025) (Mural).
